main.cpp

```diff
@@ -3,28 +3,28 @@
 
 void Result(Fighters& winner) 
 {
-	std::cout <<winner.getID() << " wins. Remaining HP: " << winner.getHP() << "\n";
+	std::cout <<winner.getID() << " wins. Remaining HP: " << winner.getHP() << ". Level: " << winner.getLevel() << "\n";
 }
 
-int main(int argc, char *argv[])
+int main(int argc, char* argv[])
 {
- 	try
-	{  
-		Fighters A = Fighters::parseUnit(argv[1]);
-		Fighters B = Fighters::parseUnit(argv[2]);
+	(void)argc;
+		try
+		{  
+			Fighters A = Fighters::parseUnit(argv[1]);
+			Fighters B = Fighters::parseUnit(argv[2]);
 			
-		while (A.getHP() > 0)
+			while (A.getHP() > 0)
 			{
  				A.Fight(B);
    				if (B.getHP() == 0) {Result(A); break;}
 				B.Fight(A);
 				if (A.getHP() == 0) {Result(B); break;}
 			}
-   	}
-	catch(const std::exception& e)
-	{
-		std::cerr << e.what() << '\n';
-	}
-
-return 0;
+   		}
```

Comment:
Most a Fighton belül is van egy ciklus, amiben oda-vissza ütik egymást, meg itt is. Csak egy helyen szerepeljen ez a loop.